validator/utils.go

```diff
@@ -0,0 +1,20 @@
+package validator
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+func SpawnerSupportsModule(spawner ValidationSpawner, requested common.Hash) bool {
+	supported, err := spawner.WasmModuleRoots()
+	if err != nil {
+		log.Warn("WasmModuleRoots returned error", "err", err)
+		return false
```

Comment:
This case is a bit concerning to me because we'll somewhat silently permanently disable this validator. Shouldn't we return an error here?

Reply:
I changed scemantics to only require one validation to succeed per wasmModuleRoot. So if one spawner is down for some reason but others can cover for it - it'll work which seems like the right behavior to me. If no spawner is found for a certain root it will error out in a different pace.